packages/nc-gui/components/smartsheet/details/Fields.vue

```diff
@@ -814,14 +814,16 @@ const onFieldOptionUpdate = () => {
                         'text-brand-500': compareCols(field, activeField),
                       }"
                     />
-                    <span
-                      :class="{
-                        'text-brand-500': compareCols(field, activeField),
-                      }"
-                      class="truncate max-w-64"
-                    >
-                      {{ fieldState(field)?.title || field.title }}
-                    </span>
+                    <a-tooltip :title="fieldState(field)?.title || field.title">
```

Comment:
Use `NcTooltip` component, `packages/nc-gui/components/nc/Tooltip.vue`

Reply:
**Fixed in below commit**

[fix : Used NcTooltip](https://github.com/nocodb/nocodb/pull/7114/commits/9207eda4c98368d9b747fb3ccff4a37b70b60fc7)
[9207eda](https://github.com/nocodb/nocodb/pull/7114/commits/9207eda4c98368d9b747fb3ccff4a37b70b60fc7)